harness/determined/pytorch/_pytorch_context.py

```diff
@@ -57,4 +85,311 @@ def get_lr_scheduler(self) -> Optional[pytorch.LRScheduler]:
             * ``optimizer()``
             * ``create_lr_scheduler()``
         """
-        return self.lr_scheduler
+        # TODO(DET-3267): deprecate this when releasing pytorch flexible primitives.
+        check.lt_eq(len(self.lr_schedulers), 1)
+        if len(self.lr_schedulers) == 1:
+            return self.lr_schedulers[0]
+        return None
+
+    def _Model(self, model: torch.nn.Module) -> torch.nn.Module:
+        check.is_none(self.model, "Must call Model() only once and before calling configure_amp.")
```

Comment:
blocking: I would expand this error message explaining to users how to they use multiple models via subclassing or point to docs at least. I would drop the configure_amp part of this message.